Revisions including reorganizing slides, correcting URLs and adding s…

reverest66 opened the pull request:
I have revised this module by adding speaker notes, organizing the slides a little bit better and confirming that the URLs provided are correct.

burtlo commented:
I feel strongly that if any one of us decides to **add**, **delete**, or **re-arrange** slides it is important to explain why and include images of the slides in the affected change. This will make it far easier to review the material without us having to blindly merge it.

> It will also make it easier to create a document that describes the differences to ourselves and our partners.

reverest66 commented:
Okay ­ I will create a readme with this information.

From:  Franklin Webber <notifications@github.com>
Reply-To:  "chef-training/chef-fundamentals-4.x"
<reply+00c29a3531a467d08b4f4f530c5f800af97a0f9d1aeb0cb192cf00000001122d19b09
2a169ce0692ac7c@reply.github.com>
Date:  Wednesday, October 7, 2015 at 10:34 AM
To:  "chef-training/chef-fundamentals-4.x"
<chef-fundamentals-4.x@noreply.github.com>
Cc:  Rebekah Everest <reverest@chef.io>
Subject:  Re: [chef-fundamentals-4.x] Revisions including reorganizing
slides, correcting URLs and adding s (#94)


I feel strongly that if any one of us decides to add, delete, or re-arrange
slides it is important to explain why and include images of the slides in
the affected change. This will make it far easier to review the material
without us having to blindly merge it.
> 
> It will also make it easier to create a document that describes the
> differences to ourselves and our partners.


Reply to this email directly or view it on GitHub
<https://github.com/chef-training/chef-fundamentals-4.x/pull/94#issuecomment
-146271312> .

reverest66 commented:
I made the following changes:

1.  I deleted the Objectives slide, because this is an informational module, not a practice/teaching module.  We don't expect them to meet an objectives - we are just providing the learners with resources.

2.  I modified the "What do I do now" test to read "Going Forward".  "What do I do now" makes the learner sound lost; conversely "Going Forward" means that the learner is moving toward something.

3.  Each resource type was preceded by an "Objectives" slide checking stuff off - like what we did in previous modules.  However, this is a different type of module without learning objectives, so I modified this to act more as a sub-header slide organizing groups that coincide with the resource type that flow as we are presenting.  (e.g., Practice Chef, Resources You Can Read, Resources You Can Watch, Audio Resources, etc.)

4.  Some of the slides were text heavy. For example, the ChefConf and Chef Summit slides were all text and not very aesthetically pleasing.  I moved the text into the speaker notes and added images to the Chef Conf slide and the community summit slide.  These images will have to be updated as time goes on, but I have a feeling we will revisit this deck frequently.  Following is an example.

![image](https://cloud.githubusercontent.com/assets/12753461/10345988/b30e06ee-6ce1-11e5-87df-243a9728dbd8.png)

Thanks!
